docs/api.md

```diff
@@ -0,0 +1,31 @@
+## Error responses
+
+The rageshake server will respond with a specific JSON payload on error:
+
+```json
+{
+    "error": "A human readable error string.",
+    "errcode": "RS_UNKNOWN",
+    "policy_url": "https://github.com/matrix-org/rageshake/blob/master/docs/blocked_rageshake.md"
+}
+```
+
+Where the fields are as follows:
+
+ - `error` is an error string to explain the error, in English.
+ - `errcode` is a machine readable error code which can be used by clients to give a localized error.
+ - `policy_url` is an optional URL that links to a reference document, which may be presented to users.
+
+### Error codes
+
+- `RS_UNKNOWN` is a catch-all error when the appliation does not have a specific error.
```

Comment:
is there an intention that "RS_" be a namespace for server errors, rather than the user-defined ones in `rejection_conditions`? It might be good to mention that explicitly.

Reply:
oh, given your rageshake.sample.yaml, I guess not.

Maybe there *should* be a separate namespace?

Or if not, the prefix seems a bit redundant?

Reply:
I think there should be a separate namespace, let's do that. Something like `RS_REJECTED_YOUR_REASON_GOES_HERE`

Reply:
Documented and tested for during config load :+1: